### PR TITLE
feat: configurable date formats and translations

### PR DESCRIPTION
**Pull request by haringsrob**

### CODE REVIEW

**Comment by faustbrian** [comment]
Hey, sorry for the late check-up. Didn't get a notification 😅

File: src/Enums/CalendarViews.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BombenProdukt\LivewireCalendar\Enums;
+
+enum CalendarViews: string
```
Comment (faustbrian) [inline src/Enums/CalendarViews.php:7]:
```suggestion
enum CalendarView: string
```

**Review by faustbrian — CHANGES_REQUESTED** [review]

File: resources/views/calendar.blade.php [hunk 2]
```diff
@@ -188,61 +188,21 @@ class="absolute right-0 z-10 mt-3 w-32 origin-top-right overflow-hidden rounded-
...
+                                    role="menuitem"
+                                    tabindex="-1"
+                                    wire:click="$set('selectedView', '{{$view->value}}')"
+                                    @click="isOpen = false"
+                                    @class([
+                                        'bg-gray-100 text-gray-900' => $selectedView === $view->value,
+                                        'block w-full px-4 py-2 text-sm text-gray-700 text-left',
+                                    ])
+                                >
+                                    {{$view->getViewLabel()}}
```
Comment (faustbrian) [inline resources/views/calendar.blade.php:203]:
```suggestion
                                    {{ $view->getViewLabel() }}
```

File: resources/views/calendar.blade.php [hunk 3]
```diff
@@ -188,61 +188,21 @@ class="absolute right-0 z-10 mt-3 w-32 origin-top-right overflow-hidden rounded-
...
-                                ])
-                            >
-                                Year View
-                            </button>
+                            @foreach(\BombenProdukt\LivewireCalendar\Enums\CalendarViews::cases() as $view)
+                                <button
+                                    type="button"
+                                    role="menuitem"
+                                    tabindex="-1"
+                                    wire:click="$set('selectedView', '{{$view->value}}')"
```
Comment (faustbrian) [inline resources/views/calendar.blade.php:196]:
```suggestion
                                    wire:click="$set('selectedView', '{{ $view->value }}')"
```

**Review by faustbrian — CHANGES_REQUESTED** [review]

**Comment by faustbrian** [comment]
@haringsrob released in 2.3.0

**Comment by haringsrob** [comment]
Thanks @faustbrian, sorry I did not fix the issues was busy with other stuff.

**Pull request merged**